Persephone/MPDClient/Extensions/MPDClient+Album.swift

```diff
@@ -49,27 +54,32 @@ extension MPDClient {
     }
   }
 
-  func allAlbums() {
+  func albums(filter: String) {
     var albums: [MPDAlbum] = []
-    var artist: String = ""
 
-    mpd_search_db_tags(self.connection, MPD_TAG_ALBUM)
-    mpd_search_add_group_tag(self.connection, MPD_TAG_ALBUM_ARTIST)
+    mpd_search_db_songs(self.connection, false)
+    if filter != "" {
```

Comment:
Does this need to account for a filter value that is only whitespace?

Reply:
Probably but the interface gives you fast enough feedback that I can't imagine it being a problem.